cmd/node/node.go

```diff
@@ -280,6 +280,34 @@ func (app *App) introduction() {
 	log.Info("Welcome to Spacemesh. Spacemesh full node is starting...")
 }
 
+type clockErrorDetails struct {
+	Drift time.Duration
+}
+
+func (c *clockErrorDetails) MarshalLogObject(encoder log.ObjectEncoder) error {
+	encoder.AddDuration("drift", c.Drift)
+	return nil
+}
+
+type clockError struct {
+	err     error
+	details clockErrorDetails
+}
+
+func (c *clockError) MarshalLogObject(encoder log.ObjectEncoder) error {
+	encoder.AddString("code", "ERR_CLOCK_DRIFT")
```

Comment:
maybe codes can be defined in a separate module, but this is not important for now i guess

Reply:
@dshulyak Do you think such a module should be inside the `log` package?